bin/boilerplate/_config.yml

```diff
@@ -16,8 +16,24 @@ title: "Lesson Title"
 # Life cycle stage of the lesson
 # See this page for more details: https://cdh.carpentries.org/the-lesson-life-cycle.html
 # Possible values: "pre-alpha", "alpha", "beta", "stable"
+#
+# Lessons that are going through the transition to the
+# Carpentries Workbench will go through 3 steps:
+# 'transition_step_1': notice indicating a new version
+# 'transition_step_2': notice encouraging to use new version
+# 'transition_step_3': notice indicating the lesson is deprecated,
```

Comment:
To keep things consistent with "pre-alpha", I would use kebab case here, not snake case:

`transition_step_1` -> `transition-step-1`